### PR TITLE
Draft of front end architecture in contributing

### PR DESCRIPTION
**Pull request by msecret**
Based on front end architecture review done, https://docs.google.com/document/d/1fHMQ_YcKP7JEQ1dO9oxESw1qZdRiXqWJ30v1wKfdmBg/edit#.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,52 @@
 This is a repo to design a beta version of 18f.gsa.gov based on the U.S.
 Web Design Standards and with a complete refresh of much of the content.
 
+## Front end architecture
+### CSS
+- Styling will be built from the US Web Design Standards.
```
Comment (maya) [inline CONTRIBUTING.md:6]: Change to "Draft U.S.   ..."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,52 @@
...
+- CSS methodology will be inherited from the WDS, which inherits mostly from the [18f front end guide](https://pages.18f.gov/frontend/css-coding-styleguide/architecture/).
```
Comment (maya) [inline CONTRIBUTING.md:7]: Consider WDS --> Draft Standards (this is how we've formally shortened it in the past)

Comment (maya) [inline CONTRIBUTING.md:7]: 18f --> 18F

Comment (maya) [inline CONTRIBUTING.md:7]: Wonder if it should be "Front end guide," but we can verify with a content person.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,6 +1,52 @@
...
+- jQuery will be required for the WDS.
+- No heavy JS framework is required.
+- Should use the AirBnB linter configuration for JavaScript, if linting fails, it will also fail the tests, but not the build.
+- Ruby gems will be used for front end dependency management.
+- JavaScript will not be bundled.
+- There will be a manual testing script that will be required to walk through for each PR.
+
+
+### Devices
+-  All versions of IE that Microsoft supports (IE9 and up); newest Chrome/FF.
```
Comment (maya) [inline CONTRIBUTING.md:37]: Let's write out Firefox.

File: CONTRIBUTING.md (hunk 2 above)
Comment (msecret) [inline CONTRIBUTING.md:7]: I've been doing WDS everywhere. I'd prefer to just use that.

Comment (maya) [inline CONTRIBUTING.md:7]: cool sounds good

**Comment by maya** [comment]
LGTM.

**Pull request merged**